gui/options.cpp

```diff
@@ -2476,6 +2477,83 @@ void GlobalOptionsDialog::addAccessibilityControls(GuiObject *boss, const Common
 }
 #endif // USE_TTS
 
+struct ExistingSave {
+	MetaEngine *metaEngine;
+	Common::String target;
+	SaveStateDescriptor desc;
+
+	ExistingSave(MetaEngine *_metaEngine, const Common::String &_target, const SaveStateDescriptor &_desc) :
+		metaEngine(_metaEngine),
+		target(_target),
+		desc(_desc)
+	{}
+};
+
+bool GlobalOptionsDialog::updateAutosavePeriod(int newValue) {
+	const int oldAutosavePeriod = ConfMan.getInt("autosave_period");
+	if (oldAutosavePeriod != 0 || newValue <= 0)
+		return true;
+	typedef Common::Array<ExistingSave> ExistingSaveList;
+	ExistingSaveList saveList;
+	using Common::ConfigManager;
+	const ConfigManager::DomainMap &domains = ConfMan.getGameDomains();
+	const Common::String autoSaveName = Common::convertFromU32String(_("Autosave"));
```

Comment:
Is this variable actually used anywhere?

Reply:
Good point. Leftovers from previous revisions.